website/docs/cloud-docs/workspaces/settings/run-tasks.mdx

```diff
@@ -20,7 +20,7 @@ You can manage run tasks through the UI or through the [Run Tasks APIs](/cloud-d
 
 ## Creating a Run Task
 
-Refer to [Run Tasks Integration](/cloud-docs/integrations/run-tasks#hcp-packer-run-task) for a list of available run tasks. These include the HCP Packer integration and run tasks developed by HashiCorp Technology Partners.
+You can access a complete list of all run tasks in the [Terraform Registry](https://registry.terraform.io/browse/run-tasks). These run tasks include the [HCP Packer integration]((/cloud-docs/integrations/run-tasks#hcp-packer-run-task) and run tasks developed by HashiCorp Technology Partners.
```

Comment:
The HCP Packer integration is not actually listed in the registry? Does that make this sentence slightly misleading? We could get it added if needed.

Reply:
oh!! I'm so so sorry, I assumed it was! I think we should add it, since it's a legitimate option for users. Thoughts?

Reply:
is that the only hashicorp created run task or are there others? I don't have access to TFC instance where I can see the currently listed run tasks. I think the answer is likely, yes we can add it.

Reply:
Nice! @gautambaghel do you have thoughts on this? We currently advertise the HCP Packer run task on the integrations page, so that's why I was thinking it might be a good idea to add it to the registry with the others.

Reply:
Yes, omg skipped my brain. we should totally add Packer run tasks on the registry. @appilon -- do you need help from me to do this?

Reply:
As discussed, just add them to the spreadsheet.

Reply:
@appilon @gautambaghel Do we know if the HCP Packer run task (or any other HashiCorp run task) has been added to the registry?